### PR TITLE
NRL pub release

### PR DESCRIPTION
**Pull request by aburrell**
# Description

Updates acknowledgements and headers to include the NRL pub release

# Type of change

Please delete options that are not relevant.

- This change requires a documentation update

# How Has This Been Tested?

unit tests

## Test Configuration
* Operating system: OS X Big Sur
* Version number: Python 3.9
* Any details about your local setup that are relevant

# Checklist:

- [x] Make sure you are merging into the ``develop`` (not ``main``) branch
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have linted the files updated in this pull request
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] Add a note to ``CHANGELOG.md``, summarizing the changes

If this is a release PR, replace the first item of the above checklist with the
release checklist on the pysat wiki:
https://github.com/pysat/pysat/wiki/Checklist-for-Release

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ Change Log
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](https://semver.org/).
 
-[0.1.X] - 2023-XX-XX
+[0.2.0] - 2024-01-10
```
Comment (aburrell) [inline CHANGELOG.md:6]:
```suggestion
[0.2.0] - 2024-XX-XX
```

File: pyproject.toml [hunk 2]
```diff
@@ -4,7 +4,7 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "pysatMadrigal"
-version = "0.1.0"
+version = "0.2.0"
```
Comment (aburrell) [inline pyproject.toml:7]: In prep for upcoming release candidate.

File: ACKNOWLEDGEMENTS.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+Funding
+=======
+The following institutions, missions, and programs have provided funding
+for pysatSpaceWeather development.
+
+Institutions
+------------
+ - Cosmic Studio
```
Comment (jklenzing) [inline ACKNOWLEDGEMENTS.md:8]: @JonathonMSmith shows up on the contributor's list, so we may need CUA here too.

Comment (aburrell) [inline ACKNOWLEDGEMENTS.md:8]:
```suggestion
- The Catholic University of America (CUA)
 - Cosmic Studio
```

**Pull request merged**